src/v2/components/core/Layout/UserInfo/ClaimButton.tsx

```diff
@@ -1,26 +1,42 @@
 import { motion } from "framer-motion";
-import React from "react";
+import React, { MouseEvent, useCallback } from "react";
 import { T } from "src/renderer/i18n";
 import { styled } from "src/v2/stitches.config";
 
 const Button = styled(motion.button, {
   appearance: "none",
   backgroundColor: "#dc9c2d",
+  border: "none",
   padding: "5px 1rem",
   "&:disabled": {
     backgroundColor: "$gray",
+    color: "white",
   },
 });
 
+const transifexTags = "v2/ClaimButton";
+
 interface ClaimButtonProps {
   onClick: () => void;
   loading: boolean;
 }
 
 export function ClaimButton({ loading, onClick }: ClaimButtonProps) {
+  const eventListener = useCallback<(e: MouseEvent) => void>(
+    (e) => {
+      e.stopPropagation();
```

Comment:
This prevents triggering the monster collection by disabling the event bubbling-up, as the parent launches the monster collection when clicked. Check [this website](https://domevents.dev/) to learn more about this.